cronjob.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+ name: hello-cronjob
+ namespace: mateapp
+spec:
+ schedule: "*/4 * * * *"
+ concurrencyPolicy: Replace
+ startingDeadlineSeconds: 100
+ successfulJobsHistoryLimit: 10
+ failedJobsHistoryLimit: 5
+ jobTemplate:
+   spec:
+     template:
+       spec:
+         containers:
+           - name: todoapp
+             image: ikulyk404/busyboxplus:curl
+             command:
+               - /bin/sh
+               - -c
+               - curl http://172.18.0.4:30008/api/health
```

Comment:
The CronJob is missing resource requests and limits for the container. It's important to specify these to ensure the job runs efficiently and does not consume excessive resources. Consider adding `resources` with `requests` and `limits` for `cpu` and `memory`.